src/neva/pal_service/agl/protos/applauncher.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
```

Comment:
Is there any way we can request the proto from AGL instead of copying hte proto here?

Reply:
I checked applaunchd repo and recipe, right now the proto file isn't installed, it's just used by applaunchd to generate code for its service implementation, the other clients (flutter and libqtappfw) also copied applauncher.proto